s3cmd.1

```diff
@@ -539,6 +539,12 @@ Silence output on stdout
 Path to SSL CA certificate FILE (instead of system
 default)
 .TP
+\fB\-\-ssl\-cert\fR=CRT_FILE
+Path to client SSL certificate CRT_FILE
+.TP
+\fB\-\-ssl\-key\fR=KEY_FILE
+Path to client SSL certificate key KEY_FILE
+.TP
```

Comment:
I think that maybe it will help the understanding of novice users to modify the help messages to:
> Path to client own SSL certificate CRT_FILE
> Path to client own SSL certificate private key KEY_FILE

Reply:
OK